Gulpfile.js

```diff
@@ -183,6 +194,25 @@ gulp.task('watch', function() {
   gulp.watch('src/js/ripple/*', [ 'build-debug' ]);
 });
 
+// To use this, each javascript file must have /* @flow */ on the first line
+gulp.task('typecheck', function() {
```

Comment:
The full flow workflow isn't integrated yet, but the transitional workflow is:
1. Add /* @flow */ to the top of a file you are working on
2. Run "gulp typecheck" and fix all the warnings/errors for that file
Once we do this for all the files, we can disable "weak mode" and do a second round of error fixing. Once that is done, we can start adding type annotations and start using the "gulp strip" task.